build-tools/packages/build-infrastructure/mocha-multi-reporter-config.json

```diff
@@ -0,0 +1,9 @@
+{
+	"reporterEnabled": "xunit,spec",
+	"xunitReporterOptions": {
+		"output": "nyc/junit-report.xml"
+	},
+	"mochaJsonOutputReporterReporterOptions": {
```

Comment:
Tony recently [removed this from the repo](https://github.com/microsoft/FluidFramework/pull/22735) so this can be cleaned up.